docs/guides/component-testing/angular/examples.mdx

```diff
@@ -193,6 +193,192 @@ in the future
+  myTitlePropAsSignal.set('FooBar')
+})
+
+cy.get('the-title-element').should('have.text', 'FooBar')
+```
+
+###### Change spy for signal `input()`
+
+Since the prop in question is an `input()`, Cypress does not propagate changes to the `titleChange` spy.
+
+```typescript
+cy.mount(TestComponent, {
+  componentProperties: {
+    title: 'Test Component',
+    // @ts-expect-error
+    titleChange: createOutputSpy('titleChange'),
+  },
+})
+
+// some action that changes the title prop...
+
+// this assertion will not be true. Cypress will never emit change events for props that are `input()` signals
+cy.get('@titleChange').should('have.been.called')
+```
+
+#### Testing Model Signals
+
+`model()` signals are writable signals, meaning they have the ability to support two-way data binding.
+
+##### Infered Generic Type
+
+In the example below, the `count` prop being passed into our `TestComponent` is the generic type of the `model()` signal. In this example, that generic type is a `number`. Cypress sets the generic value in the writable `model()`.
+
+```ts
+cy.mount(TestComponent, {
+  componentProperties: {
+    count: 1,
+  },
+})
+
+cy.get('the-count-element').should('have.text', '1')
+```
+
+##### Writable Signal
+
+Since the infered generic type in our example above lives outside the Angular context, there is no way for Cypress to enforce the databinding on context updates. In other words, updates to the `count` variable will not update the prop being passed into `TestComponent`. This is incredibly similar to how infered generic types of `input()` signals work.
+
+To solve this problem, Cypress allows users to pass a writable `signal()` wrapped in the generic type as the prop. Since this is a `model()` signal, this allows us to test two-way databinding.
+
+```typescript
+let count = signal(5)
+cy.mount(TestComponent, {
+  componentProperties: {
+    count,
+  },
+})
+
+// some action that changes the count to 7 inside the component
+
+cy.then(() => {
+  // this assertion will be true
+  expect(count()).to.be(7)
+})
+```
+
+```typescript
+let count = signal(5)
+cy.mount(TestComponentComponent, {
+  componentProperties: {
+    count,
+  },
+})
+
+cy.then(() => {
+  count.set(8)
+})
+
+// this assertion will be true
+cy.get('the-count-element').should('have.text', '8')
+
+// later in the test, some action that changes the count to 14 inside the component
+// this assertion will be true
+cy.get('the-count-element').should('have.text', '14')
+```
+
+##### Change spy for models
+
+Since the prop in question is an `model()` and is a `WritableSignal`, Cypress **will** propagate changes to `countChange` if the output spy is created, either like the example below or if `autoSpyOutputs: true` is configured. However, the typing for `countChange` are not supported since this is technically not a prop (`@Output()` actually adds this as a prop which is not true in our case, even though the signal output is completely valid).
```

Comment:
```suggestion
Since the prop in question is a `model()` and is a `WritableSignal`, Cypress **will** propagate changes to `countChange` if the output spy is created, either like the example below or if `autoSpyOutputs: true` is configured. However, the typing for `countChange` are not supported since this is technically not a prop (`@Output()` actually adds this as a prop which is not true in our case, even though the signal output is completely valid).
```

Reply:
addressed in 1055c7068